data/cardano-data.cabal

```
@@ -0,0 +1,49 @@
+cabal-version:       >=1.10
+
+name:                cardano-data
+version:             0.1.0.0
+synopsis:            Usefull datatypes for cardano libraries
+-- description:
+-- bug-reports:
+license:             Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+author:              IOHK Formal Methods Team
+maintainer:          formal.methods@iohk.io
```

I don't think the formal methods team wrote these :slightly_smiling_face:. I would use the ones from `cardano-prelude`:
```suggestion
author:              IOHK
maintainer:          operations@iohk.io
```

---

fixed